docs/grid_exchange_formats/ucte/export.md

```diff
@@ -2,6 +2,17 @@
 
 <span style="color: red">TODO</span>
 
```

Comment:
I am not sure yet how to structure documentation.
For now, I just added this small documentation to keep track of how everything is exported and once I have a better idea of how everything is done, I can maybe restructure it a bit better.